packages/client/src/utils/localStorageUtils.ts

```diff
@@ -0,0 +1,9 @@
+const USER_NAME = 'userName';
```

Comment:
maybe `/utils/storage.ts`. Both because saying utils again it too much, but also because localStorage is an implementation detail that we might change later.